### ISSUE TITLE
Unable to access to hosts in HA subnet and use HA as exit node

### ISSUE THREAD

**Issue by Mincka** [hassio-addons/addon-tailscale#12]
# Problem/Motivation

I am unable to access hosts in the HA subnet over IPv4 or access to the Internet using HA as exit node.
I am using the Android client but also tried with a Windows client.
I am using HA OS (HA Blue). 
Both access to local network and "exit node" work fine with WireGuard addon (off course disabled when using Tailscale addon).

## Expected behavior

Reaching hosts without Tailscale such as router or printer should work when subnet routing is enabled.

HA is on 10.0.0.9. My router is on 10.0.0.1, a printer is on 10.0.0.5, both expose web interface on port 80.

## Actual behavior

I get ERR_CONNECTION_TIMED_OUT error on Edge browser on Android and Windows.

I can see this in the addon logs:
```
2021/08/20 12:09:06 Accept: TCP{100.73.161.118:59894 > 10.0.0.9:80} 60 tcp ok
2021/08/20 12:09:06 Accept: TCP{100.73.161.118:59896 > 10.0.0.9:80} 60 tcp ok
2021/08/20 12:09:06 Accept: TCP{10.0.0.9:80 > 100.73.161.118:59894} 40 ok out
2021/08/20 12:09:29 Accept: TCP{100.73.161.118:38272 > 10.0.0.9:8123} 60 tcp ok
2021/08/20 12:09:29 Accept: TCP{100.73.161.118:38274 > 10.0.0.9:8123} 60 tcp ok
2021/08/20 12:11:12 Accept: TCP{100.73.161.118:41080 > 10.0.0.1:80} 60 tcp ok
2021/08/20 12:11:19 Accept: TCP{100.73.161.118:41080 > 10.0.0.1:80} 60 tcp ok
2021/08/20 12:12:28 Accept: TCP{100.73.161.118:53026 > 10.0.0.5:80} 60 tcp ok
2021/08/20 12:12:28 Accept: TCP{100.73.161.118:53028 > 10.0.0.5:80} 60 tcp ok
```

HA access over 10.0.0.9:8123 works as expected, we can see the `ok out` but not for other hosts.

I am also unable to access to the Internet with exit node enabled, no logs in the addon at all.
I also tried to stop my AdGuard addon, same behavior.

Suspicious addon's logs:
```
2021/08/20 11:49:52 router: failed to delete legacy rule, continuing anyway: checking for [-m comment --comment tailscale -i tailscale0 -j ACCEPT] in filter/FORWARD: running [/sbin/iptables -t filter -C FORWARD -m comment --comment tailscale -i tailscale0 -j ACCEPT --wait]: exit status 2: iptables v1.8.7 (legacy): Couldn't load match `comment':No such file or directory
Try `iptables -h' or 'iptables --help' for more information.
2021/08/20 11:49:52 router: failed to delete legacy rule, continuing anyway: checking for [-m comment --comment tailscale -o eth0 -j MASQUERADE] in nat/POSTROUTING: running [/sbin/iptables -t nat -C POSTROUTING -m comment --comment tailscale -o eth0 -j MASQUERADE --wait]: exit status 2: iptables v1.8.7 (legacy): Couldn't load match `comment':No such file or directory
Try `iptables -h' or 'iptables --help' for more information.
```

```
2021/08/20 11:49:52 control: controlclient: restarting map request for "router" health change to new state: adding [-i tailscale0 -j MARK --set-mark 0x40000] in v4/filter/ts-forward: running [/sbin/iptables -t filter -A ts-forward -i tailscale0 -j MARK --set-mark 0x40000 --wait]: exit status 2: iptables v1.8.7 (legacy): unknown option "--set-mark"
Try `iptables -h' or 'iptables --help' for more information.
2021/08/20 11:49:52 health("router"): error: adding [-i tailscale0 -j MARK --set-mark 0x40000] in v4/filter/ts-forward: running [/sbin/iptables -t filter -A ts-forward -i tailscale0 -j MARK --set-mark 0x40000 --wait]: exit status 2: iptables v1.8.7 (legacy): unknown option "--set-mark"
Try `iptables -h' or 'iptables --help' for more information.
```

## Steps to reproduce

Enable all subnets routes and exit node. (I also tried without the exit node disabled) and attempt to access to a host of the HA subnet with another Tailscale client.

**Comment by frenck** [comment]
Fixed in #17

**Comment by yousaf465** [comment]
how to get iptables version?
